stable/awsebscsiprovisioner/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
I don't think this should be included...

Reply:
ok, np.